atlasdb-dbkvs/src/main/java/com/palantir/atlasdb/keyvalue/dbkvs/impl/oracle/OracleQueryFactory.java

```diff
@@ -373,15 +276,16 @@ protected FullQuery getRowsColumnRangeSubQuery(
     @Override
     protected FullQuery getRowsColumnRangeFullyLoadedRowsSubQuery(
             List<byte[]> rows, long ts, ColumnRangeSelection columnRangeSelection) {
-        String query = " /* GET_ROWS_COLUMN_RANGE_FULLY_LOADED_ROWS (" + tableName + ") */ "
```

Comment:
Don't duplicate the query "name" - the wrapQueryWithIncludeValue wraps these inner queries, and we only need to specify the name there - once.